renovate.json

```diff
@@ -0,0 +1,20 @@
+{
+  "$schema": "https://docs.renovatebot.com/renovate-schema.json",
+  "extends": [
+    "local>mitodl/.github:renovate-config"
+  ],
+    "packageRules": [
+    {
+      "groupName": "all non-major dev-dependencies",
+      "groupSlug": "all-minor-dev-patch",
+      "matchPackagePatterns": [
+        "*"
+      ],
+      "matchPaths": ["test*requirement*.txt", "dev*requirement*.txt", "tst*require*.txt"],
+      "matchUpdateTypes": [
+        "minor",
```

Comment:
would changing this to "major" significantly reduce the # PR's?

Reply:
It's possible. The bigger impact would be if we had a lock-file format for Python so that changes within the range of our constraints would be done atomically. I'm working on a doc to propose a combined set of standards for our development experience which includes Renovate and Poetry to combine those benefits.